java/runtime/src/main/java/io/ray/runtime/runner/RunManager.java

```diff
@@ -84,7 +84,7 @@ public void terminateProcess(String name, Process p) {
       }
       numAttempts++;
     }
-    LOGGER.info("Process {} is now terminated.", name);
+    LOGGER.debug("Process {} is now terminated.", name);
```

Comment:
I think these logs are useful and need to be info. Why change them to debug?

Reply:
I feel the complete opposite to be honest. They've given me a really hard time seeing the actual errors buried in the middle (in fact for a long time I didn't even realize there might be errors in the middle). The non-Java jobs aren't nearly this verbose... compare [this](https://travis-ci.com/github/ray-project/ray/jobs/318669233) with [this](https://travis-ci.com/github/ray-project/ray/jobs/318669229) for instance.

(I wouldn't be surprised if this is one reason the Java builds have broken in the past by the way—the easier it is to overlook errors and the harder it is to find them, the more likely it is the build will break...)